packages/core/src/presets/orion.ts

```
@@ -0,0 +1,177 @@
+      SUPPORTED_RESOURCES: supportedResources,
+    };
+  }
+
+  static async generateAddons(
+    userData: UserData,
+    options: Record<string, any>
+  ): Promise<Addon[]> {
+    // url can either be something like https://torrentio.com/ or it can be a custom manifest url.
+    // if it is a custom manifest url, return a single addon with the custom manifest url.
+    if (options?.url?.endsWith('/manifest.json')) {
+      return [this.generateAddon(userData, options, [])];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)